vivo-project/VIVO
# [i18n-3762] Merging language specific home artifacts into VIVO

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: [3762](https://github.com/vivo-project/VIVO/issues/3762)

[Companion PR](https://github.com/vivo-project/Vitro/pull/348)

# What does this pull request do?
Moving language specific home artifacts from VIVO-languages into VIVO (+ adoption of pom files)

# What's new?
home directories from VIVO-languages repository are not needed any more. The next step is to do the same for webapp directories from VIVO-languages. After that, the VIVO-languages repositories might be archived. 

# How should this be tested?

- Checkout https://github.com/chenejac/VIVO-languages/tree/remove_language_home_artifacts as VIVO-languages project
- Remove VIVO home and install the new, fresh instance of VIVO
- Validate it is working as expected
- Try switching between all 8 languages (set in the runtime.properties languages.selectableLocales = en_US, de_DE, sr_Latn_RS, ru_RU, fr_CA, en_CA, es, pt_BR)
- Search [VIVO-home] for xcfv$This - it should return no results (this text is specified under a comment in all n3/ttl files in VIVO-languages from the first point, for instance - https://github.com/chenejac/VIVO-languages/blob/remove_language_home_artifacts/en_US/home/src/main/resources/rdf/i18n/en_US/display/firsttime/PropertyConfig_en_US.nt) 

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
